docs/docs/persistent-volumes.md

```diff
@@ -1,86 +1,303 @@
 ---
-title: Stateful Applications Using Persistent Volumes
----
+title: Stateful Applications Using Local Persistent Volumes
+-----------------------------------------------------------
```

Comment:
@sascala more than three dashes break the syntax – the included text will **not** be added to the html page as metadata, but will be rendered as text.